ui/pages/create-account/connect-hardware/index.scss

```
@@ -186,7 +186,7 @@
   }
 
   &__item:hover {
-    background-color: rgba(0, 0, 0, 0.06);
+    background-color: var(--color-overlay-default);
```

We need to add a pattern for tables so we don't really have this. Let's update the table to not have alternating backgrounds and instead add a bottom border to `&__item`
```
  &__item {
    @include Paragraph;

    flex-direction: row;
    display: flex;
    padding-left: 10px;
    padding-right: 10px;
    border-bottom: 1px solid var(--color-border-muted); // Adding border here
  }
```

Then on hover we can use background-alternative..

```suggestion
    background-color: var(--color-background-alternative);
```

---

Then we can remove these styles

```
  &__item:nth-of-type(even) {
    background-color: var(--color-background-default);
  }
  &__item:nth-of-type(odd) {
    background: var(--color-background-alternative);
  }
```